src/ex1/styles/ids.css

```diff
@@ -0,0 +1,59 @@
+#add-todo {
+  display: flex;
+}
+
+#new-todo-title:focus {
+  border: 2px solid var(--very-dark-green);
+  border-radius: 4px;
+  outline: none;
```

Comment:
it's a good practice to put all the different variants of a style after the base is defined so the variants will get a priority and could override the base class with ease and without the need for !important

Reply:
I think I got it. I will use the base class if when have another input, because for now it is the only input, with id, so I do not feel I have to overwrite something, because I do not have the need for !important